templates/report_template.html

```diff
@@ -12,34 +12,103 @@
 					<th style="text-align: center;">Last Build Number</th>
 					<th style="text-align: center;">Compose</th>
 					<th style="text-align: center;">Result</th>
-					<th style="text-align: center;">Bugzilla</th>
-					<th style="text-align: center;">Jira</th>
-					<th style="text-align: center;">Other</th>
+					<th style="text-align: center;">Blockers</th>
 				</tr>
 			</thead>
 			<tbody>
 				{% for row in rows %}
 				<tr>
+					<!-- OSP Version and Job Name -->
 					<td style="text-align: center;">{{row.osp_version}}</td>
 					<td style="text-align: left;"><a href="{{row.job_url}}">{{row.job_name}}</a></td>	
 
+					<!-- remaining columns (LCB Num, Compose, Result, Blockers) all dependent on result value -->
 					<!-- LCB is "SUCCESS" -->
 					{% if row.lcb_result == "SUCCESS" %}
 						<td style="text-align: center;"><a href="{{row.lcb_url}}">{{row.lcb_num}}</a></td>
 						<td style="text-align: center;">{{row.compose}}</td>
 						<td style="text-align: center;" bgcolor="#4646ff"><a href="https://www.youtube.com/watch?v=68ugkg9RePc" style="text-decoration: none; color: #000;">{{row.lcb_result}}</a></td>
+						<td style="text-align: center;"><p>N/A</p></td>
 
 					<!-- LCB is "UNSTABLE" -->
 					{% elif row.lcb_result == "UNSTABLE" %}
 						<td style="text-align: center;"><a href="{{row.lcb_url}}">{{row.lcb_num}}</a></td>
 						<td style="text-align: center;">{{row.compose}}</td>
 						<td style="text-align: center;" bgcolor="#fdd500">{{row.lcb_result}}</td>
+						<td>
+							<p style="text-align: center;"><b>Bugs</b></p>
+							{% for bug in row.bugs %}
+								<ul>
+								{% if bug.bug_url != None %}
+									<li><a href="{{bug.bug_url}}">{{bug.bug_name}}</a></li>
+								{% else %}
+									<li>{{bug.bug_name}}</li>
+								{% endif %}
+								</ul>
+							{% endfor %}
+							<hr>
```

Comment:
This new implementation makes the report longer, even when there are no bugs or tickets or other trackers. I suggest creating adding these conditionally, so you can reduce the noise... 


```
<td>
{% bugs = [b for b in row.bugs if b.url != None] %}
{% if bugs %}
<p style="text-align: center;"><b>Bugs</b></p>
<ul>
{% for bug in bugs %}
<li><a href="{{bug.bug_url}}">{{bug.bug_name}}</a></li>
{% endfor %}
</ul>
{% endif %}
....
```

Reply:
Interesting - could you show a sample report with this implementation so we can visualize how it differs?

Reply:
@gouthampacha I'm going to go ahead and merge this, but I would like to explore this suggestion in a future patch. Will touch base with you when I am back next week.